RELEASE_NOTES.md

```diff
@@ -1,5 +1,91 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes/> for the official Chef release notes.
 
+# What's New in 16.3
+
+## Chef InSpec 4.22.1
+
+Chef InSpec has been updated from 4.21.1 to 4.22.1. This new release includes the following improvements:
+
+- The `=` character is now allowed for command line inputs
+- `apt-cdrom` repositories are now skipped when parsing out the list of apt repositories
+- Faulty profiles are now reported instead of causing a crash
+- Errors are no longer logged to stdout with the `html2` reporter
+- macOS Big Sur is now correctly identified as macOS
+
+## New Resources
+
+### windows_firewall_profile
+
+The `windows_firewall_profile` allows you to `enable`, `disable`, or `configure` Windows Firewall profiles. For example you can now setup default actions and configure rules for the `Public` profile using this single resource instead of managing your own PowerShell code in a `powershell_script` resource:
+
+```ruby
+windows_firewall_profile 'Public' do
+  default_inbound_action 'Block'
+  default_outbound_action 'Allow'
+  allow_inbound_rules 'False'
+  display_notification 'False'
```

Comment:
The final version of this had this using `true` and `false` rather than the Strings.  Though I realize I didn't update the examples as I thought I had, so I'm putting in a PR to fix that now, but it should be fixed here as well.

Reply:
PR #10198

Reply:
If you left the properties empty wouldn't you get just an enable?

Reply:
Fixed